hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1966,6 +1966,14 @@
       jar and false for the ozone-filesystem-lib-current.jar
     </description>
   </property>
+  <property>
+    <name>ozone.fs.iterate.batch-size</name>
+    <value>1</value>
+    <tag>OZONE, OZONEFS</tag>
+    <description>
+      iterate batch size of delete and rename when use BasicOzoneFileSystem.
```

Comment:
Can we update the title of the JIRA to reflect batch rename?

Reply:
I can modify the description here until rename's PR commits.